### ISSUE TITLE
3.0 - Orders - Inputs Missing Labels

### ISSUE THREAD

**Issue by spencerfinnell** [easydigitaldownloads/easy-digital-downloads#7737]
### Expected behavior

When Adding or Viewing an Order, any `input`s should have a linked label.

### Actual behavior

Many inputs use a `span` instead of a true `label`.

### Steps to reproduce the behavior
1) Visit Orders > Add New
2) Click the "Key:" label.

Notice the input is not focused.

### Information (if a specific version is affected):
EDD Version (or branch): `release/3.0`

Any other relevant information:

Changes should be made against [`try/edd3-orders`](https://github.com/easydigitaldownloads/easy-digital-downloads/tree/try/edd3-orders) which includes some layout changes.

**Comment by LisaCee** [comment]
I didn't link it right, but I've started working on this [here](https://github.com/easydigitaldownloads/easy-digital-downloads/tree/issue/7737).
